spec/controllers/api/v1/project_preferences_controller_spec.rb

```diff
@@ -137,4 +137,34 @@
 
     it_behaves_like "is creatable"
   end
+
+  describe "#update_settings", :focus do
+    let!(:project) { create(:project, owner: authorized_user) }
+    let!(:upp) { create(:user_project_preference, project: project) }
+    let(:settings_params) do
+      {
+        user_id: upp.user_id,
+        project_id: project.id,
+        settings: { workflow_id: 1234 }
+      }
+    end
+
+    let(:scopes) { %w(project_preference) }
+    before(:each) do
+      default_request user_id: authorized_user.id, scopes: scopes
+    end
+
+    it "responds with a 200" do
+      post :update_settings, settings_params
+      expect(response.status).to eq(200)
+    end
+
+    xit "updates the UPP settings attribute" do
```

Comment:
not a typo, just skipping the test because the "successful request" one is still broken.